apps/manager_app/device_authentication.c

```diff
@@ -116,54 +116,50 @@ typedef enum {
  */
 static pb_size_t get_request_type(const manager_auth_device_request_t *request);
 
+/**
+ * @brief Sends response of type manager_auth_device_response_t to the host. It
+ * internally encodes the response to type manager_result_t
+ *
+ * @param resp Reference to the response to be send to the host.
+ */
+static void send_auth_device_response(manager_auth_device_response_t *resp);
+
 /**
  * @brief This API sends the response for device authentication flow completion
  * to the host.
  *
  */
-static manager_auth_device_response_t send_flow_complete(void);
+static void send_flow_complete(void);
 
 /**
  * @brief This function is the request handler for the SIGN_SERIAL_NUM state of
  * the device authentication flow
  *
  * @param query Reference to query of type manager_query_t as received from the
  * host
- * @param result Reference to the result of type manager_result_t, which will be
- * populated by the API. This API will only fill the auth_device field of the
- * manager_result_t.
  * @return device_auth_state_e The next state of the flow
  */
-static device_auth_state_e sign_serial_handler(const manager_query_t *query,
-                                               manager_result_t *result);
+static device_auth_state_e sign_serial_handler(const manager_query_t *query);
```

Comment:
in all the 3 handlers, add some explanation in doxygen comments for each of the query types and their expected behavior.

Reply:
I have added required comments for full picture